DigitalLearningSolutions.Web/Helpers/ExternalApis/MapsApiHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers.ExternalApis
+{
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.Models.External.Maps;
+    using Newtonsoft.Json;
+
+    public interface IMapsApiHelper
+    {
+        Task<MapsResponse> GetPostcodeCoordinates(string postcode);
+    }
+
+    public class MapsApiHelper : IMapsApiHelper
+    {
+        private readonly HttpClient client;
+        private readonly string mapsApiBaseUrl;
+
+        public MapsApiHelper(HttpClient httpClient)
+        {
+            var mapsApiKey = ConfigHelper.GetAppConfig()["MapsAPIKey"];

```

Comment:
I wonder if it's worth putting the "MapsAPIKey" string into ConfigHelper and add a method "GetMapsApiKey" into the config helper so if we ever change the key name, we'll only need to change it in two places.. 🤔